src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -35,204 +31,211 @@
-
-            JsonNode jsonNode = mapper.readValue(jsonNodeNew.toString(), JsonNode.class);
-            ObjectNode objNode = (ObjectNode) jsonNode;
-            objNode.set("aggregatedObject", mapper.readTree(object));
-
-            return jsonNode;
-        } catch (Exception e) {
-            log.error(e.getMessage(), e);
-        }
-        return null;
-    }
-
-    public JsonNode getAggregatedObject(String dbDocument) {
-        ObjectMapper mapper = new ObjectMapper();
-        try {
-            JsonNode documentJson = mapper.readValue(dbDocument, JsonNode.class);
-            JsonNode objectDoc = documentJson.get("aggregatedObject");
-            return objectDoc;
-        } catch (Exception e) {
-            log.error(e.getMessage(), e);
-        }
-        return null;
-    }
-
-    public String extractObjectId(JsonNode aggregatedDbObject) {
-        return aggregatedDbObject.get("_id").textValue();
-    }
-
-    /**
-     * Locks the document in database to achieve pessimistic locking. Method
-     * findAndModify is used to optimize the quantity of requests towards
-     * database.
-     * 
-     * @param id
-     *            String to search
-     * @return String aggregated document
-     */
-    public String lockDocument(String id) {
-        boolean documentLocked = true;
-        String conditionId = "{\"_id\" : \"" + id + "\"}";
-        String conditionLock = "[ { \"lock\" :  null } , { \"lock\" : \"0\"}]";
-        String setLock = "{ \"$set\" : { \"lock\" : \"1\"}}";
-        ObjectMapper mapper = new ObjectMapper();
-        while (documentLocked == true) {
-            try {
-                JsonNode documentJson = mapper.readValue(setLock, JsonNode.class);
-                JsonNode queryCondition = mapper.readValue(conditionId, JsonNode.class);
-                ((ObjectNode) queryCondition).set("$or", mapper.readValue(conditionLock, JsonNode.class));
-                Document result = mongoDbHandler.findAndModify(databaseName, collectionName, queryCondition.toString(),
-                        documentJson.toString());
-                if (result != null) {
-                    log.debug("DB locked by " + Thread.currentThread().getId() + " thread");
-                    documentLocked = false;
-                    return JSON.serialize(result);
-                }
-                // To Remove
-                log.debug("Waiting by " + Thread.currentThread().getId() + " thread");
-            } catch (Exception e) {
-                log.error(e.getMessage(), e);
-            }
-        }
-        return null;
-    }
-
-    public int getTtl() {
-        int ttl = 0;
-        if (ttlValue != null && !ttlValue.isEmpty()) {
-            try {
-                ttl = Integer.parseInt(ttlValue);
-            } catch (NumberFormatException e) {
-                log.error(e.getMessage(), e);
-            }
-        }
-        return ttl;
-    }
+	static Logger log = (Logger) LoggerFactory.getLogger(ObjectHandler.class);
```

Comment:
What have changed here? nothing? Is it some IDE tools that change formatting that cases this.
Easier to review if you use this autoformatting tool after review, since that save time for the reviewers.

Reply:
I have auto-formatted the code in eclipse...